docs/07-Advanced-Features/Fuel-Related/Charge-Temp-Estimation.md

```diff
@@ -1 +1,39 @@
-# Charge Temperature Estimation  
\ No newline at end of file
+# Charge Temperature Estimation
+
+Charge temperature estimation is used to model the increase in intake air temperature after the IAT sensor by the heat of the intake piping, intake manifold, cylinder head, etc. Without accurate charge temperature estimation it is possible that an engine may exhibit leaner air fuel ratios at higher IATs, especially under low load like idle and light cruise. This is common when the VE table is tuned at lower IATs without charge temperature estimation.
+
+## Mode
+
+Three modes of charge temperature estimation are available; _RPM+TPS_, _Air Mass Interpolation_ and _Table_.
+
+### RPM+TPS (Default)
+
+Interpolates between four coefficients based on RPM and TPS to generate a coefficient for estimating the intake air temperature. The coefficient determines the percentage of the estimated air temperature that comes from the IAT with the remainder coming from the CLT.
+
+![image](https://github.com/user-attachments/assets/1eecfe48-18ae-4510-8b28-ca4c4c882153)
```

Comment:
@DenisMitchell, please commit these image asset files to the repository and reference them here.  In other instances we just create a directory sibling to the page and toss assets in there.
![image](https://github.com/user-attachments/assets/2fd4243f-5d4d-4408-8ce1-a0d8f7cd53c3)

Reply:
Thanks, I followed the example for Boost-Control and updated the PR.